data/index.js

```diff
@@ -27,6 +27,7 @@ function setDefaults(benefit, institution) {
   benefit.id = benefit.id || benefit.slug
   benefit.top = benefit.top || top
   benefit.floorAt = benefit.floorAt || 1
+  benefit.imgSrc = benefit.imgSrc?.slice("img/".length)
```

Comment:
```suggestion
  benefit.imgSrc = benefit.imgSrc?.replace(/^img\//i, '')
```
Ça évite les edge-cases où l'url ne commencerait pas par `img/`

Reply:
En effet, j'en profite pour le changer à endroit ou j'ai trouvé ce bout de code